tv_apps/browser/js/sync/bookmarks.js

```diff
@@ -70,31 +70,28 @@ var SyncBookmark = (function () {
     });
   }
 
-  function init() {
-    syncDataStore = new SyncDsHelper('sync_bookmarks_store');
-    return syncDataStore.init();
-  }
-
   function start() {
-    if (!syncDataStore) {
-      return Promise.reject('Uninitialized DataStore');
+    if (syncDataStore) {
+      return Promise.resolve();
     }
-    syncDataStore.registerStoreChangeEvent(() => {
-      syncDataStore.dataStoreSync(handleTask);
+    // XXX Should not we be using bookmarks_store?
+    syncDataStore = new SyncDsHelper('sync_bookmarks_store');
```

Comment:
It will be modified [here](https://github.com/weilonge/gaia/commit/1bdf197aea11533b15233640a131822c549960f9#diff-21d7ad0430c83716cd9435b01780c6c8R87).